.github/workflows/build_notarized.yml

```diff
@@ -276,7 +272,14 @@ jobs:
       with:
         submodules: recursive
         ref: ${{ inputs.branch || github.ref_name }}
-        sparse-checkout: .github
+        sparse-checkout: |
+          .github
+          Gemfile
+          Gemfile.lock
+          fastlane
```

Comment:
These files are needed in every sparse checkout for fastlane to work, obviously.